tezos/interop/src/lib.rs

```diff
@@ -23,6 +23,8 @@ mod tezos_ffi {
     use crate::TzResult;
 
     ocaml! {
+        pub fn dump(a: ());
```

Comment:
Let's move these (and the wrappers) to their own module (`tezos_fuzzing_ffi` or something like that) that is guarded by the fuzzing feature flag you use